Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -107,11 +106,18 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
     }
     
     open func attach(to parentView: UIView, controller: UIViewController) {
+        guard let containerView = activeContainer?.view else {
+            Logger.logError("Active container should not be nil when attaching to parent view", scope: "Core")
+            return
+        }
```

Comment:
Is there a direct way to cover this scenario with unit test?

Reply:
This scenario is an aberration. Meaning that although it is possible in the compiler context, on the application context we **always** initialize a Core with a Container at the CoreFactory. IMO, `activeContainer` as an optional property shouldn't even exist.

Having said that, I don't think there's an actual gain in testing it, as it won't ever happen 🤷🏽‍♂️

Reply:
I tend to think about tests in a different way. As much as the responsibility for creating these objects lies with other entities, it is the responsibility of Core to ensure that these objects exist. He doesn't know the moment they are created. Another point is that where these objects are created they can be modified and the `Core` can no longer have this security and break because there is no `activeContainer` at that moment.
So I see the creation of this test as a gain.
Makes sense?

Reply:
It really does. We added the test on 1926bc87bf79ed653cb3d5860f028d398884aa4a